CONTRIBUTING.md

```diff
@@ -127,9 +127,15 @@ export PATH="/opt/homebrew/opt/libtool/libexec/gnubin:$PATH"
 
 ### Build
 
+Clone the repo and make sure to include the submodules by adding `--recurse-submodules`. For example:
+
+```shell
+git clone --recurse-submodules https://github.com/runfinch/finch.git
```

Comment:
is it worth adding a note to tell users to run `git submodule update --init` if they cloned without `--recurse-submodules`? the error when you try to build without submodules is a bit obtuse:
```
go build -ldflags "-X github.com/runfinch/finch/pkg/version.Version=903a1d4 -X github.com/runfinch/finch/pkg/version.GitCommit=903a1d49aa0a41bef7bac8160a42ec995b69d629" -o /Users/bernings/scratch/runfinch/finch-test/_output/bin/finch github.com/runfinch/finch/cmd/finch
cd deps/finch-core && \
		FINCH_OS_x86_URL="" \
		FINCH_OS_AARCH64_URL="" \
		VDE_TEMP_PREFIX=/Users/bernings/scratch/runfinch/finch-test/_output/dependencies/vde/opt/finch \
		/Library/Developer/CommandLineTools/usr/bin/make
make[1]: *** No targets specified and no makefile found.  Stop.
make: *** [finch-core] Error 2
```

Reply:
I think CONTRIBUTING.md makes more sense. Honestly, it might be worth adding the note right below this line and before the `make` command.